day-1/files/staging/hetzner-staging-1/promtail-config.yaml

```diff
@@ -0,0 +1,22 @@
+server:
+  http_listen_port: 9080
+  grpc_listen_port: 0
+
+positions:
+  filename: /tmp/positions.yaml
+
+clients:
+  - url: "https://loki.rpch.tech/loki/api/v1/push"
+
+scrape_configs:
+  - job_name: system
+    docker_sd_configs:
+      - host: unix:///var/run/docker.sock
+        refresh_interval: 5s
+    pipeline_stages:
+      - static_labels:
+          hostname: "hetzner-staging-alpha-1"
```

Comment:
I think that for the sake of readability, it would be helpful to use the ansible task `template` and modify this file as a jinja2 template and only modify the attribute `hostname`